docs/user-guide.md

```diff
@@ -383,8 +384,8 @@ The following example shows how to select a subgraph based upon vertex and edge
 
 <div data-lang="scala"  markdown="1">
 {% highlight scala %}
-import org.graphframes.examples
-val g: GraphFrame = examples.Graphs.friends
+import org.graphframes._
```

Comment:
generally spark follows the guideline of not import ._ unless there are > 5 classes/types

Reply:
Ok, then I'll put explicit imports into it.